<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление записи в таблицу</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background: qlineargradient(
        x1: 0, y1: 0,       /* верхний левый угол */
        x2: 1, y2: 1,       /* нижний правый угол */
        stop: 0 #1e3c72,    /* насыщенный синий */
        stop: 0.5 #2a5298,  /* бирюзовый */
        stop: 1 #1abc9c     /* зелёный, ассоциирующийся с природой */
    );
    color: white;           /* цвет текста для контраста */
    font-family: &quot;Segoe UI&quot;, Tahoma, Geneva, Verdana, sans-serif;
    font-size: 14pt;

</string>
  </property>
  <widget class="QWidget" name="widget_dialog" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0.6);           /* затемнённый полупрозрачный фон */
    border: 1px solid rgba(255, 255, 255, 0.8);     /* тонкая, но чёткая рамка */
    border-radius: 12px;
    padding: 15px;</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_dialog"/>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_dialog">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>450</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
    border: none;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_save">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #3498db;       /* насыщенный синий */
    color: white;                    /* белый текст */
    border: 2px solid white;         /* белый край */
    border-radius: 8px;              /* скруглённые углы */
    padding: 6px 12px;               /* уменьшенные внутренние отступы (сверху/снизу и слева/справа) */
    font-size: 16px;
    font-weight: 600;
    min-width: 80px;                 /* уменьшенная минимальная ширина */
    transition: background-color 0.3s ease;
}

QPushButton:hover {
    background-color: #2980b9;       /* чуть темнее при наведении */
    border: 2px solid white;         /* белый край при наведении */
}

QPushButton:pressed {
    background-color: #1c5980;       /* ещё темнее при нажатии */
    border: 2px solid white;         /* белый край при нажатии */
}
</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #3498db;       /* насыщенный синий */
    color: white;                    /* белый текст */
    border: 2px solid white;         /* белый край */
    border-radius: 8px;              /* скруглённые углы */
    padding: 6px 12px;               /* уменьшенные внутренние отступы (сверху/снизу и слева/справа) */
    font-size: 16px;
    font-weight: 600;
    min-width: 80px;                 /* уменьшенная минимальная ширина */
    transition: background-color 0.3s ease;
}

QPushButton:hover {
    background-color: #2980b9;       /* чуть темнее при наведении */
    border: 2px solid white;         /* белый край при наведении */
}

QPushButton:pressed {
    background-color: #1c5980;       /* ещё темнее при нажатии */
    border: 2px solid white;         /* белый край при нажатии */
}
</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
